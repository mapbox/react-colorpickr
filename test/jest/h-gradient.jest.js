import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as testCases from '../test-cases/h-gradient-test-cases';

describe('HInput', () => {
  let testCase;
  let wrapper;

  describe(testCases.inactive.description, () => {
    beforeEach(() => {
      testCase = testCases.inactive;
      wrapper = shallow(React.createElement(testCase.component, testCase.props));
    });

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe(testCases.active.description, () => {
    beforeEach(() => {
      testCase = testCases.active;
      wrapper = shallow(React.createElement(testCase.component, testCase.props));
    });

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
