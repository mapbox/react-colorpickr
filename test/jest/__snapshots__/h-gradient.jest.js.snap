// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HInput inactive renders 1`] = `<noscript />`;

exports[`HInput inactive renders 2`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "blue",
      }
    }
  />
  <div
    style={Object {}}
  />
  <div
    style={Object {}}
  />
</div>
`;
